workflows/sarek/main.nf

```diff
@@ -149,8 +156,59 @@ workflow SAREK {
 
         // Figure out if input is bam or fastq
         input_sample_type = input_sample.branch{
-            bam:   it[0].data_type == "bam"
-            fastq: it[0].data_type == "fastq"
+            bam:                 it[0].data_type == "bam"
+            fastq_gz:            it[0].data_type == "fastq_gz"
+            one_fastq_gz_spring: it[0].data_type == "one_fastq_gz_spring"
+            two_fastq_gz_spring: it[0].data_type == "two_fastq_gz_spring"
```

Comment:
why are the spring fles split, but not the normal fastq files?

Reply:
Good question. Again, the variables could perhaps have been named more appropriately. Anyways, here is my attempt at explaining what is going on:

"one_fastq_gz_spring" is when you have just one spring-file containing both R1 and R2. (In that case, the spring should be listed under `spring_1` in the samplesheet.)

"two_fastq_gz_spring" is when you have two spring-files containing R1 and R2, respectively. (In that case, the R1-spring and R2-spring should be listed under `spring_1` and `spring_2`, respectively.)

For fastq.gz-input there is always two files.